Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-Description: ASIM Windows Event Registry Creation Event Parser (event number 4657)
+Description: |
+        This ASIM parser supports normalizing Microsoft Windows Event logs, produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema (event number 4657).
 ParserName: vimRegistryEventMicrosoftWindowsEvent
 ParserQuery: | 
               let RegistryType = datatable (TypeCode:string,TypeName:string)
```

Comment:
EventProduct should 'Security Events'

Reply:
What is the rule we establish here? maybe the product for WEF should be the (etw) provider?

Reply:
We decided that the "Type" field refers to the table the data is stored in, while Product and Vendor reflect the true source.